hyrax/README.md

```diff
@@ -4,26 +4,23 @@
 The application contains 
 * Hyrax version 2.3
 * Rails 5.1
-* Ruby version 2.4
+
+Requires:
+* redis
+* Ruby 2.3.1 - 2.5.3 (as tested)
```

Comment:
I think Hyrax is tested with ruby 2.3, 2.4 and 2.5. You do not need to mention the minor version of ruby.

Also it requires Rails 5 and bundler (as you figured out below)